corehq/form_processor/submission_post.py

```diff
@@ -571,3 +514,104 @@ def notify_submission_error(instance, message, exec_info=None):
         notify_exception(request, message, details=details, exec_info=exec_info)
     else:
         logging.error(message, exc_info=sys.exc_info(), extra={'details': details})
+
+
+class SubmissionPostFormProcessor(object):
```

Comment:
maybe rename to `DefaultFormProcessor`

Reply:
there are already classes `FormProcessorCouch`, `FormProcessorSQL`, `FormProcessorInterface` and I didn't want to add another name that looks like that. Hence the use of `SubmissionPost` in the name to signify that this is related for `SubmissionPost`.
Can use `SubmissionPostDefaultFormProcessor` or `SubmissionPostDefaultProcessor` if any of those sound better.

Reply:
hmm, how about `FormXmlProcessor`?

Reply:
Updated.